iosHyperskillApp/fastlane/Fastfile

```diff
@@ -239,7 +239,13 @@ platform :ios do
       cocoapods(try_repo_update_on_error: true)
     end
 
-    scan()
+    scan(
+        clean: true,
+        skip_build: true,
+        skip_package_dependencies_resolution: true,
+        device: "iPhone SE (3rd generation)",
```

Comment:
Let's also try `prelaunch_simulator: true` parameter. Please compare several execution times with it and without